.github/workflows/CI.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+    tags: ['*']
+  pull_request:
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - 'min'
+          - '1'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+        include:
+          - os: windows-latest
+            version: '1'
+          - os: macos-latest
+            version: '1'
+    steps:
+      - uses: actions/checkout@v6
+      - uses: julia-actions/cache@v2
+      - uses: julia-actions/setup-julia@v2
+        with:
+          version: ${{ matrix.version }}
+      - uses: julia-actions/julia-runtest@v1
+      - uses: julia-actions/julia-processcoverage@v1
+        with:
+          directories: src
```

Comment:
This is more restrictive than the default ([`src,ext`](https://github.com/julia-actions/julia-processcoverage/blob/ecc9c0165392fbdca296e17219ee4406a8388aa8/action.yml#L13)), why even bothering setting it?
```suggestion
```